doc/radosgw/nfs.rst

```diff
@@ -103,7 +103,7 @@ following characteristics:
 
     * additional RGW authentication types such as Keystone are not currently supported
 
-Manually configuring an NFS-Ganesha Instance
+Manually Configuring an NFS-Ganesha Instance
```

Comment:
Please check if there are references / links to this section and below anywhere in the docs, I don't know if they're case-sensitive.

Reply:
Thank you a good point, checked and none exist:
```console
$ fgrep -Ri 'Manually Configuring an NFS-Ganesha Instance' doc/
doc/radosgw/nfs.rst:Manually Configuring an NFS-Ganesha Instance

```